src/status.rs

```diff
@@ -353,7 +353,7 @@ impl Status {
                             "modified:" => DiffType::Modified,
                             "renamed:" => DiffType::Renamed,
                             "deleted:" => DiffType::Deleted,
-                            _ => panic!("Unknown prefix: `{prefix}`"),
+                            _ => return Err(Error::msg("Unknown prefix: `{prefix}`")),
```

Comment:
For both cases, I would prefer a slightly more descriptive error message than the original panic provided. For example:

```suggestion
                            _ => return Err(Error::msg("unknown prefix in `git diff` output: `{prefix}`")),
```

*note: it is also deliberate that the U has become lowercase, for stylistic consistency when the error is printed*